integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -99,7 +139,7 @@ context('Prisoner Overview page - Education And Training tab', () => {
         .activeTabIs('Education and training')
         .doesNotHaveFunctionalSkillsDisplayed()
         .hasCuriousUnavailableMessageDisplayed()
-        .hasCompletedInPrisonQualificationsDisplayed()
+        .hasCompletedInPrisonQualificationsLast12MonthsDisplayed()
```

Comment:
Likewise - if the Curious learner profile API as returned a 500 (which is what this test is), how are we able to display the table showing courses completed in the last 12 months?
Why are we not displaying the "curious is unavailable message" in place of the completed courses table?